modules/apps/portal-remote/portal-remote-rest-extender/src/main/java/com/liferay/portal/remote/rest/extender/internal/CXFJaxRsServiceRegistrator.java

```diff
@@ -91,37 +93,43 @@ private void _addService(Object service) {
 	}
 
 	private void _registerApplication(Bus bus, Application application) {
-		RuntimeDelegate runtimeDelegate = RuntimeDelegate.getInstance();
+		try (SafeCloseable safeCloseable = ThreadContextClassLoaderUtil.swap(
+				AggregateClassLoader.getAggregateClassLoader(
+					RuntimeDelegateImpl.class.getClassLoader(),
```

Comment:
Hi Dante, 
As I can see while debugging, the classLoader of RuntimeDelegateImpl is from "shared-dependencies-cxf", and by using the AggregateClassLoader of RuntimeDelegateImpl and current class, it can successfully get a RuntimeDelegateImpl from shared-dependencies-cxf whild doing `RuntimeDelegate.getInstance()`